### PR TITLE
Issue #3: Hyperlink styles are inconsistent

### PR DESCRIPTION
**Pull request by adrianbanks**
I've made all the hyperlinks underlined, except those in the menu.

### CODE REVIEW

File: DDDEastAnglia/Content/Site.css [hunk 1]
```diff
@@ -7,11 +7,9 @@ dd {
...
-.span1 > p > a, .span2 > p > a, .span3 > p > a, .span4 > p > a,
-.span5 > p > a, .span6 > p > a, .span7 > p > a, .span8 > p > a,
-.span9 > p > a, .span10 > p > a, .span11 > p > a, .span12 > p > a {
-    text-decoration: underline;
-}
+a { text-decoration: underline; }
+
+a.btn, .dropdown-menu a { text-decoration: none; }
 
 .carousel-caption a { color: white; text-decoration: underline; }
```
Comment (alastairs) [inline DDDEastAnglia/Content/Site.css]: You could delete the `text-decoration` style from this class too.

...and I still need to regenerate the base Bootstrap stylesheet :worried:

File: DDDEastAnglia/Content/Site.css [hunk 2]
```diff
@@ -7,13 +7,13 @@ dd {
...
 }
 
-.span1 > p > a, .span2 > p > a, .span3 > p > a, .span4 > p > a,
-.span5 > p > a, .span6 > p > a, .span7 > p > a, .span8 > p > a,
-.span9 > p > a, .span10 > p > a, .span11 > p > a, .span12 > p > a {
-    text-decoration: underline;
-}
+a { text-decoration: underline; }
+
+a:hover { color: #008600; }
```
Comment (alastairs) [inline DDDEastAnglia/Content/Site.css:12]: How did you determine this colour?

**Comment by adrianbanks** [comment]
I took the original hyperlink colour as defined in bootstrap and lightened it (gauged by eye). Too light and it is hard to read when hovered over, too dark and you can't tell there is haver colour. That colour seemed to work well.

**Comment by alastairs** [comment]
Ok cool. It looks very similar to one of the colours I put into the Bootstrap CSS generator, so I was curious :smile:

**Pull request merged**